github_actions/spec/dependabot/pull_request_creator/labeler_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Dependabot::PullRequestCreator::Labeler do
+  subject(:labeler) do
+    described_class.new(
+      source: source,
+      credentials: [],
+      custom_labels: nil,
+      includes_security_fixes: false,
+      dependencies: [dependency],
+      label_language: true,
+      automerge_candidate: false
+    )
```

Comment:
I would love to add sane default values for most of these attributes because the constructor requires a lot of parameters and they seem to distract from the test and make it difficult to use this class for very simple setups.